client/src/helpers/APICalls/requests.ts

```diff
@@ -0,0 +1,51 @@
+import { FetchOptions } from '../../interface/FetchOptions';
+import { Request, RequestsList } from '../../interface/Request';
+import { User } from '../../interface/User';
+
+export async function getRequestList(): Promise<RequestsList> {
+  const fetchOptions: FetchOptions = {
+    method: 'GET',
+    credentials: 'include',
+  };
+
+  return await fetch('/request/', fetchOptions).then((res) =>
+    res.json().catch(() => ({ error: { message: 'Unable to connect to server. Please try again' } })),
+  );
+}
+
+export async function createRequest(
+  sitterId: User,
+  startDate: Date,
+  endDate: Date,
+  serviceType: string,
+  totalPrice: number,
+): Promise<Request> {
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ sitterId, startDate, endDate, serviceType, totalPrice }),
+    credentials: 'include',
+  };
+  return await fetch('/request/', fetchOptions).then((res) =>
+    res.json().catch(() => ({ error: { message: 'Unable to connect to server. Please try again' } })),
+  );
+}
+
+export async function updateRequest(
+  requestId: string,
+  status: string,
+  startDate?: Date,
+  endDate?: Date,
+  serviceType?: string,
+  totalPrice?: number,
+): Promise<{ request: Request }> {
+  const fetchOptions: FetchOptions = {
+    method: 'PUT',
```

Comment:
Consider using a `PATCH` request if you're not to update the whole resource.

Reply:
Yes, thanks. Same recommendation on the Request controller PR, so I changed it to PUT instead.